src/app/components/AccountDetailLayout/index.tsx

```diff
@@ -0,0 +1,176 @@
+import {
+  CaretLeftIcon,
+  CrossIcon,
+  ExportIcon,
+} from "@bitcoin-design/bitcoin-icons-react/filled";
+import Header from "@components/Header";
+import IconButton from "@components/IconButton";
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import Modal from "react-modal";
+import QRCode from "react-qr-code";
+import { Outlet, useNavigate, useParams } from "react-router-dom";
+import Avatar from "~/app/components/Avatar";
+import Loading from "~/app/components/Loading";
+import TextField from "~/app/components/form/TextField";
+import api, { GetAccountRes } from "~/common/lib/api";
+import msg from "~/common/lib/msg";
+import { Account } from "~/types";
+
+type AccountAction = Pick<Account, "id" | "name">;
+
+function AccountDetailLayout() {
+  const navigate = useNavigate();
+
+  const { t } = useTranslation("translation", {
+    keyPrefix: "accounts.account_view",
+  });
+  const [account, setAccount] = useState<GetAccountRes>();
+  const { id } = useParams() as { id: string };
+  const [exportLoading, setExportLoading] = useState(false);
+  const [exportModalIsOpen, setExportModalIsOpen] = useState(false);
+  const [lndHubData, setLndHubData] = useState({
+    login: "",
+    password: "",
+    url: "",
+    lnAddress: "",
+  });
+
+  useEffect(() => {
+    (async () => {
+      const account = await api.getAccount(id);
+      setAccount(account);
+    })();
+  }, [id, navigate]);
+
+  async function exportAccount({ id, name }: AccountAction) {
+    setExportLoading(true);
+    setExportModalIsOpen(true);
+    setLndHubData(
+      await msg.request("accountDecryptedDetails", {
+        name,
+        id,
+      })
+    );
+    setExportLoading(false);
+  }
+  function closeExportModal() {
+    setExportModalIsOpen(false);
+  }
+
+  return (
+    <>
+      <Header
+        title={t("title1")}
+        headerLeft={
+          <IconButton
+            onClick={() => navigate("/accounts")}
```

Comment:
Can we make this a back button? when I am on a "subpage" and want to go back I end up on the accounts index page again.

Reply:
Good catch. Will update :ok_hand: